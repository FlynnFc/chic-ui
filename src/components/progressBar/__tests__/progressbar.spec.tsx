import ProgressBar from '..';
import { render } from 'test/utils';

describe('ProgressBar Component', () => {
  it('Renders Correctly', () => {
    const ProgressBarGroup = render(
      <>
       <ProgressBar progress={80} type="danger" />
       <ProgressBar progress={60} type="success" showProgress={false} />
       <ProgressBar progress={40} animated />
      </>
    );
    expect(ProgressBarGroup).toMatchSnapshot();
  });
});
