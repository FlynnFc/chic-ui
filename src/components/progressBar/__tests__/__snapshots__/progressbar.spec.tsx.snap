// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-gsTEea dYAXdu"
  >
    <div
      className="sc-bdfBQB ePBJvV"
      type="danger"
      width={80}
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "height": "100%",
            "justifyContent": "center",
          }
        }
      >
        <span
          className="sc-dlfnuX eNxsCi"
          type="danger"
        >
          80
          % 
        </span>
      </div>
    </div>
  </div>,
  <div
    className="sc-gsTEea dYAXdu"
  >
    <div
      className="sc-bdfBQB gqPdhi"
      type="success"
      width={60}
    />
  </div>,
  <div
    className="sc-gsTEea dYAXdu"
  >
    <div
      className="sc-bdfBQB VfOvq"
      type="primary"
      width={40}
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "height": "100%",
            "justifyContent": "center",
          }
        }
      >
        <span
          className="sc-dlfnuX eNxsCi"
          type="primary"
        >
          40
          % 
        </span>
      </div>
    </div>
  </div>,
]
`;
